libdevcore/Exceptions.h

```diff
@@ -31,7 +31,13 @@
 namespace dev
 {
 // base class for all exceptions
-struct Exception: virtual std::exception, virtual boost::exception { mutable std::string m_message; };
+struct Exception: virtual std::exception, virtual boost::exception
+{
+	Exception(std::string _message = std::string()): m_message(_message) {}
```

Comment:
FYI msvc does not call a constructor at all for brace initialized default function arguments
http://stackoverflow.com/questions/21044488/known-compiler-bug-in-vc12